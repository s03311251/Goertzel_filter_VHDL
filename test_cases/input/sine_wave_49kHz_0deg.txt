2000
29B2
327B
3987
3E2C
3FFB
3EC8
3AB0
3415
2B97
2202
183C
0F32
07BB
028D
0024
00BA
0441
0A64
128E
1BFD
25CC
2F0F
36E8
3C99
3F9A
3FA2
3CB0
370C
2F3D
25FE
1C30
12BD
0A8A
045B
00C5
0020
0279
079A
0F06
180A
21CE
2B67
33ED
3A93
3EBA
3FFC
3E3D
39A6
32A5
29E3
2033
167E
0DAE
0698
01E4
0006
0129
0533
0BC2
1438
1DCA
2791
30A2
3822
3D5D
3FD6
3F50
3BD8
35C1
2D9F
2435
1A66
111D
093B
037D
006D
0055
0338
08D0
1095
19CE
239C
2D13
354F
3B8A
3F2E
3FE4
3D99
3886
3125
2826
1E64
14C8
0C3A
0588
0154
0001
01B1
063B
