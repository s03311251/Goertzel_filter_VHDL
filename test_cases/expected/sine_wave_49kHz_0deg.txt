FFECE720
FFEFAEA0
